applications/GeoMechanicsApplication/tests/cpp_tests/test_transient_thermal_element.cpp

```diff
@@ -241,34 +231,30 @@ void ExpectDoubleMatrixEqual(const Matrix& rExpectedMatrix, const Matrix& rActua
 void ExpectDoubleVectorEqual(const Vector& rExpectedVector, const Vector& rActualVector)
 {
     KRATOS_EXPECT_EQ(rActualVector.size(), rExpectedVector.size());
-    for (std::size_t i = 0; i < rActualVector.size(); i++)
-    {
+    for (std::size_t i = 0; i < rActualVector.size(); i++) {
         KRATOS_EXPECT_DOUBLE_EQ(rActualVector[i], rExpectedVector[i]);
     }
 }
 
 KRATOS_TEST_CASE_IN_SUITE(ThermalElement_ReturnsExpectedMatrixAndVector_WhenCalculateLocalSystem,
                           KratosGeoMechanicsFastSuiteWithoutKernel)
 {
-    Model this_model;
+    Model      this_model;
     ModelPart& model_part = this_model.CreateModelPart("Main", 3);
     model_part.AddNodalSolutionStepVariable(TEMPERATURE);
     model_part.AddNodalSolutionStepVariable(DT_TEMPERATURE);
     GenerateTransientThermalElement2D3N(model_part);
     SetupElement(model_part);
 
-    Element::Pointer p_element = model_part.pGetElement(1);
+    Element::Pointer   p_element              = model_part.pGetElement(1);
     const ProcessInfo& r_current_process_info = model_part.GetProcessInfo();
 
     Matrix left_hand_side_matrix;
     Vector right_hand_side;
-    p_element->CalculateLocalSystem(left_hand_side_matrix, right_hand_side,
-                                    r_current_process_info);
+    p_element->CalculateLocalSystem(left_hand_side_matrix, right_hand_side, r_current_process_info);
 
     Matrix expected_matrix(3, 3);
-    expected_matrix <<=  5,   -5,    0,
-                        -5,  5.5, -0.5,
-                         0, -0.5,  0.5;
+    expected_matrix <<= 5, -5, 0, -5, 5.5, -0.5, 0, -0.5, 0.5;
```

Comment:
This might be nice to revert and add the tags to ignore this for formatting

Reply:
Done.